quickstarts/Function_calling_config.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>Nit: In the set_light_color function, maybe change the output to:</p><p> </p><p>print("Light-color set to: ", rgb_hex)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done - also added `LIGHTBOT: ` prefixes to make it clear the output is from the functions, vs model response.